pkg/cli/create_helm.go

```diff
@@ -724,10 +725,6 @@ func (cmd *createHelm) getKubernetesVersion() (*version.Info, error) {
 
 		majorMinorVer := semver.MajorMinor(cmd.KubernetesVersion)
 
-		if splittedVersion := strings.Split(cmd.KubernetesVersion, "."); len(splittedVersion) > 2 {
```

Comment:
Whats the reasoning for removing this? Since it is only deprecated we still need that don't we?

Reply:
@FabioKaram, please check the associated issue. There is no point deprecating a feature that doesn't even work.

Reply:
@felipecrs yes you are right, we should remove it instead

Reply:
@FabianKramm I had thought of removing the flags but there are some other flags eg "distro" which is also not used but is deprecated, which is why I did not remove this flag. If we are okay with removing this flag and related code, I would like to remove ``distro`` flag as well? WDYT?

Reply:
@kale-amruta distro is still used internally though while kubernetes-version is not working at all anymore, so I would wait with removing distro flag.

Reply:
Ok, makes sense, I will remove just the `kubernetes-version`